### ISSUE TITLE
Feature Request - Pre-procesing

### ISSUE THREAD

**Issue by adambelis** [gwygonik/SquiggleDrawInkscape#4]
It would be handy to have some preprocessing options like contrast and brightness built in to the extension.

**Comment by gwygonik** [comment]
I probably _won't_ add this until I can figure out why the Inkscape rasterization produces odd results. (I think it probably has something to do with resolution settings, but can't verify). I'll leave this open as a reminder, and just in case 😃

**Comment by adambelis** [comment]
fair enough !  if you are interested there is a community chat for extension developers so maybe they can  help you.
https://chat.inkscape.org/channel/inkscape_extensions
